docs/bokeh/source/docs/dev_guide/setup.rst

```diff
@@ -75,6 +76,34 @@ Cloning the repository creates a ``bokeh`` directory at your file system
 location. This local ``bokeh`` directory is referred to as the *source checkout*
 for the remainder of this document.
 
+.. note::
+    Before continuing, you should make sure that all tags have been added to your
+    local directory by running the following command:
+
+    .. code-block:: sh
+
+        git tag -l |tail
```

Comment:
```suggestion
        git tag -l | tail
```

Reply:
As far as I know, `tail` isn't available on Windows, so this will only work on Linux/mac. We could leave out the `tail` command, or use `.. tab-set::` like we do in other places in this chapter!

Reply:
@tcmetzger `tail` works on Windows without problems 😊